tests/templates/kuttl/smoke/30-install-superset.yaml.j2

```diff
@@ -33,9 +33,15 @@ spec:
     vectorAggregatorConfigMapName: vector-aggregator-discovery
 {% endif %}
   nodes:
+    envOverrides:
+      COMMON_VAR: role-value # overridden by role group below
```

Comment:
nit: `COMMON_ENV_VAR` would be more speaking since we add vars for `configOverrides` too.